src/Uno.UI/UI/Xaml/Controls/Primitives/SelectorItem.cs

```diff
@@ -180,7 +181,7 @@ private void UpdateCommonStates(ManipulationUpdateKind manipulationUpdate = Mani
 				pause = false;
 			}
 
-			if (delay < TimeSpan.Zero)
+			if (delay <= TimeSpan.Zero)
```

Comment:
Why is it needed?

Reply:
It was just an invalid comparison I found (as I was forcing the delay to 0 to cause my bug :) )